### PR TITLE
Toggle was renamed to meWATCH (issue #23895)

### PR DESCRIPTION
**Pull request by xaverh**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [ ] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

Toggle.sg got renamed to meWATCH, I updated the URLs and it works again. I honestly have no idea how that flake8 thing works, though.
The patch itself is just an update to a hostnames:
video.toggle.sg is www.mewatch.sg now.

### CODE REVIEW

File: youtube_dl/extractor/mewatch.py [hunk 1]
```diff
@@ -15,11 +15,11 @@
 )
 
 
-class ToggleIE(InfoExtractor):
-    IE_NAME = 'toggle'
-    _VALID_URL = r'https?://video\.toggle\.sg/(?:en|zh)/(?:[^/]+/){2,}(?P<id>[0-9]+)'
+class MewatchIE(InfoExtractor):
+    IE_NAME = 'mewatch'
+    _VALID_URL = r'https?://www\.mewatch\.sg/(?:en|zh)/(?:[^/]+/){2,}(?P<id>[0-9]+)'
```
Comment (dstftw) [inline youtube_dl/extractor/mewatch.py]: Do not remove the old domain.

File: youtube_dl/extractor/mewatch.py [hunk 2]
```diff
@@ -15,11 +15,11 @@
 )
 
 
-class ToggleIE(InfoExtractor):
-    IE_NAME = 'toggle'
-    _VALID_URL = r'https?://video\.toggle\.sg/(?:en|zh)/(?:[^/]+/){2,}(?P<id>[0-9]+)'
+class MewatchIE(InfoExtractor):
```
Comment (dstftw) [inline youtube_dl/extractor/mewatch.py]: Do not touch this.

File: docs/supportedsites.md [hunk 3]
```diff
@@ -480,6 +480,7 @@
  - **META**
  - **metacafe**
  - **Metacritic**
+ - **meWATCH**
```
Comment (dstftw) [inline docs/supportedsites.md]: Do not touch this file.

**Comment by xaverh** [comment]
I changed it to leave the old `toggle.py` as is. Is that what you meant.

**Comment by dstftw** [comment]
No, that does not what I meant. What I meant is written in the review in english.

File: youtube_dl/extractor/mewatch.py (hunk 2 above)
Comment (xaverh) [reply youtube_dl/extractor/mewatch.py]: So, we keep the old name then?

**Pull request merged**